abstractions/php/src/RequestInfo.php

```diff
@@ -0,0 +1,19 @@
+<?php
+namespace Microsoft\Kiota\Abstractions;
+
+use Psr\Http\Message\StreamInterface;
+
+abstract class RequestInfo {
```

Comment:
the generation result instantiates the request info class, might want to remove the abstract keyword